src/main/java/org/gridsuite/sensitivityanalysis/server/service/nonevacuatedenergy/NonEvacuatedEnergyRunContext.java

```diff
@@ -34,12 +34,11 @@ public NonEvacuatedEnergyRunContext(UUID networkUuid,
                 reportInfos != null ? reportInfos : new ReportInfos(null, null, null),
                 userId,
                 provider != null ? provider : "",
-                nonEvacuatedEnergyInputData,
-                null);
+                nonEvacuatedEnergyInputData);
```

Comment:
not related to this PR but the above provider can be passed directly, set empty string as default value has no meaning.

Reply:
Done.